BookingsApi/BookingsApi.DAL/Commands/JudiciaryPersonCommandBase.cs

```diff
@@ -0,0 +1,29 @@
+using System;
+
+namespace BookingsApi.DAL.Commands
+{
+    public abstract class JudiciaryPersonCommandBase
+    {
+        protected JudiciaryPersonCommandBase(Guid externalRefId, string personalCode, string title, 
+            string knownAs, string surname, string fullname, string postNominals, string email)
+        {
+            ExternalRefId = externalRefId;
+            PersonalCode = personalCode;
+            Title = title;
+            KnownAs = knownAs;
+            Surname = surname;
+            Fullname = fullname;
+            PostNominals = postNominals;
+            Email = email;
+        }
+        
+        public Guid ExternalRefId { get; set; }
```

Comment:
We can drop the setters here since they're not being used

Reply:
Done